src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/LdapRegistrationDAO.scala

```diff
@@ -63,7 +63,10 @@ class LdapRegistrationDAO(
     }
 
   override def deleteUser(userId: WorkbenchUserId): IO[Unit] =
-    executeLdap(IO(ldapConnectionPool.delete(userDn(userId))))
+    executeLdap(for {
```

Comment:
mind to add one line comment to call out this method will do 2 things, something like (1)remove from enable user group? (2)delete user from ldap?